pkg/controller/certificaterequest/certificaterequest_controller.go

```diff
@@ -136,6 +138,20 @@ func (r *ReconcileCertificateRequest) Reconcile(request reconcile.Request) (reco
 		return reconcile.Result{}, err
 	}
 
+	// fetch the clusterdeployment and bail out if there's an outgoing migration annotation
```

Comment:
can we separate out the formatting of the cd name to make this a little more readable?